xml/Microsoft.Extensions.Primitives/StringSegment.xml

```diff
@@ -1334,8 +1334,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>Returns the <see cref="T:System.String" /> represented by this <see cref="T:Microsoft.Extensions.Primitives.StringSegment" /> or <code>String.Empty</code> if the <see cref="T:Microsoft.Extensions.Primitives.StringSegment" /> does not contain a value.</summary>
-        <returns>The <see cref="T:System.String" /> represented by this <see cref="T:Microsoft.Extensions.Primitives.StringSegment" /> or <code>String.Empty</code> if the <see cref="T:Microsoft.Extensions.Primitives.StringSegment" /> does not contain a value.</returns>
+        <summary>Returns the <see cref="T:System.String" /> represented by this <see cref="T:Microsoft.Extensions.Primitives.StringSegment" /> or <c>String.Empty</c> if the <see cref="T:Microsoft.Extensions.Primitives.StringSegment" /> does not contain a value.</summary>
+        <returns>The <see cref="T:System.String" /> represented by this <see cref="T:Microsoft.Extensions.Primitives.StringSegment" /> or <c>String.Empty</c> if the <see cref="T:Microsoft.Extensions.Primitives.StringSegment" /> does not contain a value.</returns>
```

Comment:
Why are these code rather rather see elements?

Reply:
Good point. I was mostly converting code to c elements, but here we can use the see links.